test/system-program.test.js

```diff
@@ -43,3 +49,95 @@ test('assign', () => {
   expect(transaction.programId).toEqual(SystemProgram.programId);
   // TODO: Validate transaction contents more
 });
+
+test('SystemInstruction create', () => {
+  const from = new Account();
+  const to = new Account();
+  const program = new Account();
+  const amount = 42;
+  const space = 100;
+  const recentBlockhash = 'EETubP5AKHgjPAhzPAFcb8BAY1hMH639CWCFTqi3hq1k'; // Arbitrary known recentBlockhash
+  const create = SystemProgram.createAccount(
+    from.publicKey,
+    to.publicKey,
+    amount,
+    space,
+    program.publicKey,
+  );
+  const transaction = new Transaction({recentBlockhash}).add(create);
+
+  const systemInstruction = SystemInstruction.from(transaction.instructions[0]);
+  expect(systemInstruction.fromPubkey).toEqual(from.publicKey);
```

Comment:
I just realized we're further mixing `pubkey` and `publicKey`.   `fromPublicKey` and `toPublicKey` is probably most consistent with the rest of solana-web3.js at this point.  Sorry to bike shed.

Reply:
meh 👌